### ISSUE TITLE
Management interface for users, groups, roles, permissions

### ISSUE THREAD

**Issue by nightsh** [rootio/rootio_web#232]
We need a configuration interface for (re)defining permissions, roles and groups.

- Permissions are atomic entities.
- Roles have *n* permissions and can belong to *n* groups
- Groups have *n* roles and *n* permissions
- Users are members of *n* groups

It is not necessary to define group memberships in the UI, but it would be nice to have as a feature. It also might be worth integrating the user management interface into this one, or the other way around.

Blocked by #230
